packages/ui/src/views/pull-request/pull-request-list.tsx

```diff
@@ -0,0 +1,90 @@
+import { useMemo, useState } from 'react'
+import { Link } from 'react-router-dom'
+
+import * as StackedList from '../../components/stacked-list'
+import { PullRequestListDescription } from './pull-request-list-description'
+import { PullRequestListHeaderTitle } from './pull-request-list-header-title'
+import { PullRequestListTitle } from './pull-request-list-title'
+import { PullRequestType } from './types'
+
+export interface PullRequestListPageProps {
+  pullRequests?: PullRequestType[]
+  // LinkComponent: React.ComponentType<{ to: string; children: React.ReactNode }>
+  handleResetFilters?: () => void
+  hasActiveFilters?: boolean
+  query?: string
+  handleResetQuery: () => void
+  closed_prs?: number
+  open_prs?: number
```

Comment:
Camelcase for consistency
```suggestion
  closedPRs?: number
  openPRs?: number
```